src/object/flower.cpp

```diff
@@ -26,34 +26,32 @@ Flower::Flower(BonusType _type, const std::string& custom_sprite) :
   type(_type),
   sprite(),
   flip(NO_FLIP),
-  lightsprite(SpriteManager::current()->create("images/objects/lightmap_light/lightmap_light-small.sprite"))
+  lightsprite()
 {
   m_col.m_bbox.set_size(32, 32);
-  lightsprite->set_blend(Blend::ADD);
 
   if (type == BONUS_FIRE) {
     sprite = SpriteManager::current()->create(custom_sprite.empty() ? "images/powerups/fireflower/fireflower.sprite" : custom_sprite);
     SoundManager::current()->preload("sounds/fire-flower.wav");
-    lightsprite->set_color(Color(0.3f, 0.0f, 0.0f));
   }
   else if (type == BONUS_ICE) {
     sprite = SpriteManager::current()->create(custom_sprite.empty() ? "images/powerups/iceflower/iceflower.sprite" : custom_sprite);
     SoundManager::current()->preload("sounds/fire-flower.wav");
-    lightsprite->set_color(Color(0.0f, 0.1f, 0.2f));
   }
   else if (type == BONUS_AIR) {
     sprite = SpriteManager::current()->create(custom_sprite.empty() ? "images/powerups/airflower/airflower.sprite" : custom_sprite);
     SoundManager::current()->preload("sounds/fire-flower.wav");
-    lightsprite->set_color(Color(0.15f, 0.0f, 0.15f));
   }
   else if (type == BONUS_EARTH) {
     sprite = SpriteManager::current()->create(custom_sprite.empty() ? "images/powerups/earthflower/earthflower.sprite" : custom_sprite);
     SoundManager::current()->preload("sounds/fire-flower.wav");
-    lightsprite->set_color(Color(0.0f, 0.3f, 0.0f));
```

Comment:
Where did all these lightsprite colors go?

Reply:
The file.